.github/workflows/update_yarn_lock.yaml

```diff
@@ -10,6 +9,11 @@ on:
     secrets:
       pr_token:
         required: true
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
```

Comment:
@jrafanie This one is a little interesting, because this is a workflow_call (i.e. a shared workflow), so I'm not sure if concurrency matters here?